scripted.sbt

```diff
@@ -0,0 +1,7 @@
+ScriptedPlugin.scriptedSettings
+
+scriptedLaunchOpts := { scriptedLaunchOpts.value ++
+  Seq("-Xmx1024M", "-XX:MaxPermSize=256M", "-Dplugin.version=" + version.value)
+}
+
+scriptedBufferLog := false
```

Comment:
Copy pasted from http://eed3si9n.com/testing-sbt-plugins :-)

I checked the build without this option. The build log will not print as much information then. I'd recommend we keep it.